browser-test/src/support/applicant_questions.ts

```diff
@@ -36,17 +38,18 @@ export class ApplicantQuestions {
     await this.page.waitForSelector('.cf-address-street-1 input')
     await this.page.waitForSelector('.cf-address-street-2 input')
     await this.page.waitForSelector('.cf-address-city input')
-    await this.page.waitForSelector('.cf-address-state input')
```

Comment:
Can you see if using `.cf-address-state select` will allow you to keep this line?

Reply:
Tried. It didn't work.